File: VB/dxSampleGrid/MyClasses/SomeClass.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace dxSampleGrid
	Public Class SomeClass
		Public Sub New(ByVal j As Integer)
			Id = j
			Name = "Name" & j.ToString()
		End Sub
		Private privateName As String
		Public Property Name() As String
			Get
				Return privateName
			End Get
			Set(ByVal value As String)
				privateName = value
			End Set
		End Property
		Private privateId As Integer
		Public Property Id() As Integer
			Get
				Return privateId
			End Get
			Set(ByVal value As Integer)
				privateId = value
			End Set
		End Property
	End Class
End Namespace

File: CS/dxSampleGrid/MyClasses/SomeClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace dxSampleGrid {
    public class SomeClass {
        public SomeClass(int j) {
            Id = j;
            Name = "Name" + j.ToString();
        }
        public string Name { get; set; }
        public int Id { get; set; }
    }
}
